### DISCUSSION TITLE
Disabling new registrations and google authentication

### DISCUSSION THREAD

**Discussion by axor-mst** [donetick/donetick#28] [Ideas]
Is it currently possible to disable new registrations and google authentication? If not, it would be nice to have such feature for publicly available instances, preferably controlled by environment variables.

**Comment by c0rrre** [comment]
did the login work for you? I only get invalid request messages if i try to register an account

**Comment by axor-mst** [comment]
Yes, I have succesfully registered a new account and logged in on my instance. Now, I would like to disable the possibility to register another account.

**Comment by meauxt** [comment]
@axor-mst  will look into adding this. might not be in the next release but hopefully sometime soon!

**Comment by hackmoore** [comment]
You can do this by changing the `is_user_creation_disabled` in the docker-compose.yml file (if you're using that). The option for creation still appears but hitting submit prevents creation.

**Reply by jlar0che** [reply]
This doesn't work. I added `is_user_creation_disabled = true` as an environment variable in my docker-compose.yml but I'm still able to create new users.

**Reply by jlar0che** [reply]
I found the correct way to do it. Don't try setting `is_user_creation_disabled` in your docker-compose.yml. You have to set it in your selfhosted.yaml file which lives in your /config directory.

Hope that helps someone else out there.

**Reply by ydkmlt84** [reply]
Thank you!
